app/Console/Commands/Update.php

```diff
@@ -117,12 +117,12 @@ private function migrateCollationTest()
 
         $databasename = $connection->getDatabaseName();
 
-        $schemata = $connection->table('information_schema.schemata')
-                ->select('DEFAULT_CHARACTER_SET_NAME')
-                ->where('schema_name', '=', $databasename)
-                ->get();
+        $schemata = DB::select(
```

Comment:
Not sure why we need to rewrite this part?

Reply:
Sure, because when you use the ->table method Laravel automatically appends the DB_PREFIX to the database name. The thing is that information_schema does not have a prefix as it is a common metadata table so it will try to make a query to {DB_PREFIX}information_schema.schemata which always throws an error. I rewrote it to ensure that there will not be a prefix added to this specific query only.

The same thing applies to the other location where I had this.

Reply:
OK I see! Thanks for the expanations.